src/components/prove-identity/tests/prove-identity-integration.test.ts

```diff
@@ -0,0 +1,133 @@
+import request from "supertest";
+import { describe } from "mocha";
+import { sinon } from "../../../../test/utils/test-utils";
+import nock = require("nock");
+import decache from "decache";
+import {
+  API_ENDPOINTS,
+  HTTP_STATUS_CODES,
+  PATH_NAMES,
+} from "../../../app.constants";
+import { expect } from "chai";
+import { Application } from "express";
+
+describe.only("Integration::prove identity", () => {
```

Comment:
Just spotted this .only. Best remove that before merging.

We may be best adding an eslint rule for this to the codebase, https://stackoverflow.com/a/54015366

Reply:
Ahhhhhhh!!!! I thought I'd taken these out, thanks for the spot